File: InternshipOnlineLearning/Dtos/InstructorStatsDto.cs
namespace InternshipOnlineLearning.Dtos
{
    public class InstructorStatsDto
    {
        public string Id { get; set; } = string.Empty;
        public string FullName { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public int TotalCourses { get; set; }
        public int TotalStudents { get; set; }
        public double QuizAverageScore { get; set; }
        public string TopQuizTitle { get; set; }
        public string LowQuizTitle { get; set; }
    }
}

File: Online Learning Platform/src/app/types/InstructorStatsDto.ts
// src/types/InstructorStatsDto.ts
export interface InstructorStatsDto {
  id: string;
  fullName: string;
  email: string;
  totalCourses: number;
  totalStudents: number;
  quizAverageScore: number;
  topQuizTitle: string;
  lowQuizTitle?: string;
}
